extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -2,6 +2,8 @@
 
 /* smart proxy to article comment */
 
+use Wikia\Logger\WikiaLogger;
+
 class WallMessage {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "WallMessage" has 126 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1448)